seed/static/seed/js/controllers/insights_property_controller.js

```diff
@@ -226,18 +288,18 @@ angular.module('BE.seed.controller.insights_property', [])
           }
 
           // place in appropriate dataset
-          if (_.includes($scope.data.results_by_cycles[$scope.chart_cycle]['y'], prop.property_view_id)) {
+          if (_.includes($scope.data.results_by_cycles[$scope.configs.chart_cycle]['y'], prop.property_view_id)) {
             // compliant dataset
             datasets[0]['data'].push(item);
-          } else if (_.includes($scope.data.results_by_cycles[$scope.chart_cycle]['n'], prop.property_view_id)) {
+          } else if (_.includes($scope.data.results_by_cycles[$scope.configs.chart_cycle]['n'], prop.property_view_id)) {
             // non-compliant dataset
             datasets[1]['data'].push(item);
 
             // add whisker annotation
             // only when we are displaying the non-compliant metric (energy or emission)
             // don't add whisker if data is in range for that metric or it looks bad
             let add = false
-            metric_type = $scope.chart_metric == 0 ? $scope.data.metric.energy_metric_type : $scope.data.metric.emission_metric_type;
+            metric_type = $scope.configs.chart_metric == 0 ? $scope.data.metric.energy_metric_type : $scope.data.metric.emission_metric_type;
```

Comment:
`===`, and I'm going to stop adding these comments since there are boat loads of `==` that need to be replaced